examples/apps/cchmc_ped_abd_ct_seg_app/app.py

```diff
@@ -0,0 +1,283 @@
+        self.add_flow(study_loader_op, series_selector_op, {("dicom_study_list", "dicom_study_list")})
+        self.add_flow(
+            series_selector_op, series_to_vol_op, {("study_selected_series_list", "study_selected_series_list")}
+        )
+        self.add_flow(series_to_vol_op, abd_seg_op, {("image", "image")})
+
+        # note below the dicom_seg_writer, dicom_sr_writer, dicom_sc_writer, and mongodb_entry_creator each require
+        # two inputs, each coming from a source operator
+
+        # DICOM SEG
+        self.add_flow(
+            series_selector_op, dicom_seg_writer, {("study_selected_series_list", "study_selected_series_list")}
+        )
+        self.add_flow(abd_seg_op, dicom_seg_writer, {("seg_image", "seg_image")})
+
+        # DICOM SR
+        self.add_flow(
+            series_selector_op, dicom_sr_writer, {("study_selected_series_list", "study_selected_series_list")}
+        )
+        self.add_flow(abd_seg_op, dicom_sr_writer, {("result_text_dicom_sr", "text")})
+
+        # DICOM SC
+        self.add_flow(
+            series_selector_op, dicom_sc_writer, {("study_selected_series_list", "study_selected_series_list")}
+        )
+        self.add_flow(abd_seg_op, dicom_sc_writer, {("dicom_sc_dir", "dicom_sc_dir")})
+
+        # MongoDB
+        self.add_flow(
+            series_selector_op, mongodb_entry_creator, {("study_selected_series_list", "study_selected_series_list")}
+        )
+        self.add_flow(abd_seg_op, mongodb_entry_creator, {("result_text_mongodb", "text")})
+        self.add_flow(mongodb_entry_creator, mongodb_writer, {("mongodb_database_entry", "mongodb_database_entry")})
+
+        logging.info(f"End {self.compose.__name__}")
+
+
+# series selection rule in JSON, which selects for axial CT series; flexible ST choices:
+# StudyDescription: matches any value
+# Modality: matches "CT" value (case-insensitive); filters out non-CT modalities
+# ImageType: matches value that conatins "PRIMARY", "ORIGINAL", and "AXIAL"; filters out most cor and sag views
+# SeriesDescription: matches any values that do not contain "cor" or "sag" (case-insensitive); filters out cor and sag views
+# SliceThickness: supports list, string, and numerical matching:
+# [3, 5]: matches ST values beween 3 and 5
+# "^(5(\\\\.0+)?|5)$": RegEx; matches ST values of 5, 5.0, 5.00, etc.
+# 5: matches ST values of 5, 5.0, 5.00, etc.
+# all valid series will be selected; downstream operators only perform inference and write outputs for 1st selected series
+# please see more detail in DICOMSeriesSelectorOperator
+
+Sample_Rules_Text = """
+{
+    "selections": [
+        {
+            "name": "Axial CT Series",
+            "conditions": {
+                "StudyDescription": "(.*?)",
+                "Modality": "(?i)CT",
+                "ImageType": ["PRIMARY", "ORIGINAL", "AXIAL"],
+                "SeriesDescription": "(?i)^(?!.*(cor|sag)).*$",
+                "SliceThickness": [3, 5]
```

Comment:
This range support looks good once we publish the next version MD SDK package (as the your PR had already been merged). In the meantime, the editable SDK with the recent code will be needed. Planning to do the next release of MD App SDK later in the month and bumping the version to 3.0, following the release of holoscan v3.0, but we can discuss if a minor 2.x version needs to be squeezed in.